test/dotcom_web/live/trip_planner_test.exs

```diff
@@ -78,12 +62,6 @@ defmodule DotcomWeb.Live.TripPlannerTest do
     end
 
     test "shows errors on form submit", %{view: view} do
-      html =
-        view
-        |> element("form")
-        |> render_submit()
-
-      assert html =~ "Please add a destination."
```

Comment:
📓 Same here. I think we'll want to figure out our final error display approach before writing these tests.